src/widgets/sparkline.rs

```diff
@@ -79,9 +128,43 @@ impl<'a> Sparkline<'a> {
         self
     }
 
+    /// Sets the style of the empty bars.
+    ///
+    /// `style` accepts any type that is convertible to [`Style`] (e.g. [`Style`], [`Color`], or
+    /// your own type that implements [`Into<Style>`]).
+    ///
+    /// The foreground corresponds to the bars while the background is everything else.
+    #[must_use = "method moves the value of self and returns the modified value"]
+    pub fn empty_bar_style<S: Into<Style>>(mut self, style: S) -> Self {
+        self.empty_bar_style = style.into();
+        self
+    }
+
+    /// Sets the symbol to use for empty bars.
+    ///
+    /// The default is [`EmptyBarSymbol::Empty`].
+    #[must_use = "method moves the value of self and returns the modified value"]
+    pub fn empty_bar_symbol(mut self, symbol: EmptyBarSymbol) -> Self {
+        self.empty_bar_symbol = symbol;
+        self
+    }
+
     /// Sets the dataset for the sparkline.
     ///
-    /// # Example
+    /// Each item in the dataset is a bar in the sparkline. The height of the bar is determined by
+    /// the value in the dataset and a bar may also be empty.
+    ///
+    /// The dataset can be a slice of `u64`, `Option<u64>`, or a `SparkItem`.
+    ///
+    /// If the dataset is a slice of `u64` or `Option<u64>`, the bars will be styled with the
+    /// default style of the sparkline.
+    ///
+    /// If the dataset is a slice of `SparkItem`, the bars will be styled with the style provided
+    /// in the `SparkItem`.
```

Comment:
Updated the language:

```rust
/// If the data provided is a slice of `u64` or `Option<u64>`, the bars will be styled with the
/// style of the sparkline. If the data is a slice of [`SparklineBar`], the bars will be
/// styled with the style of the sparkline combined with the style provided in the [`SparklineBar`]
/// if it is set, otherwise the sparkline style will be used.
```